source/distributed/grid_refinement.cc

```diff
@@ -257,24 +257,16 @@ namespace internal
 
                 // Count how many of our own elements would be above this
                 // threshold:
-                types::global_dof_index my_count =
+                types::global_cell_index my_count =
                   std::count_if(criteria.begin(),
                                 criteria.end(),
                                 [test_threshold](const double c) {
                                   return c > test_threshold;
                                 });
 
                 // Potentially accumulate in a 64bit int to avoid overflow:
```

Comment:
That comment is now hanging a bit -- there is no reference to anything 64-bit in the following statement, nor to overflow. Just remove it.